docs/contributing/testing.md

```diff
@@ -109,10 +109,12 @@ When you create the pull request for your contributions, list what you’ve test
 
 If your contribution changes how a component looks, include before and after screenshots if you can.
 
-### If GitHub shows the build failed
+### If GitHub shows that some checks were not successful
```

Comment:
```suggestion
### If GitHub shows that some checks failed
```

Reply:
Might be some context missing here – when the build fails the 'headline' shows 'Some checks were not successful', then lists all of the checks, with their various statuses ('Success', 'Failing')

![Screenshot 2020-12-03 at 17 18 48](https://user-images.githubusercontent.com/121939/101064212-a763e480-358b-11eb-8d52-97d2eb7a4fe6.png)

There's an example of a failing build here, if you want to try the steps: https://github.com/alphagov/govuk-frontend/pull/2053

So 'some checks were not successful' is trying to mimic the terms used in the GitHub interface. Happy to change it if you still think it makes sense to do so though!

Reply:
@36degrees That makes sense! Thanks for explaining. Sounds like the original wording is fine then. I'll delete this suggestion.